sql/src/test/java/io/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -4416,4 +4416,9 @@ private static CascadeExtractionFn CASCADE(final ExtractionFn... fns)
   {
     return Arrays.asList(aggregators);
   }
+
+  private static ExpressionPostAggregator EXPRESSION_POST_AGG(final String name, final String expression)
```

Comment:
There's a lot of other methods that are similarly named, like `DIMS`, `SELECTOR`, etc. If we want to adopt a style forbidding all-caps static methods names then I'd rather change them all in a separate patch.